app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -35,27 +46,35 @@
             android:id="@+id/back_text"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginStart="32dp"
-            android:layout_marginTop="56dp"
+            android:layout_marginStart="8dp"
             android:text="@string/back_text"
             android:textColor="@color/white_50opacity"
             android:textSize="12sp"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+            app:layout_constraintBottom_toBottomOf="@+id/back_img"
+            app:layout_constraintStart_toEndOf="@+id/back_img"
+            app:layout_constraintTop_toTopOf="@+id/back_img" />
+
+        <ImageView
+            android:id="@+id/pg_bkg"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="222dp"
+            app:layout_constraintStart_toEndOf="@+id/guideline_left"
+            app:layout_constraintTop_toTopOf="parent"
+            app:srcCompat="@drawable/ic_rectangle" />
 
         <TextView
             android:id="@+id/pg"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginTop="222dp"
-            android:background="@color/pg_color"
-            android:paddingVertical="4dp"
             android:text="@string/pg"
             android:textColor="@color/white"
             android:textSize="12sp"
```

Comment:
maybe just 

android:backgroun="@drawable/ic_rectangle"

here instead of new view?